swagger_info.go

```diff
@@ -0,0 +1,60 @@
+package swag
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"text/template"
+)
+
+// DocumentationReader is an interface to inject Swagger info reader.
+type DocumentationReader interface {
+	Swagger
+	InstanceName() string
+}
+
+// SwaggerInfo holds exported Swagger Info so clients can modify it.
+type SwaggerInfo struct {
+	Version          string
+	Host             string
+	BasePath         string
+	Schemes          []string
+	Title            string
+	Description      string
+	InfoInstanceName string
+	SwaggerTemplate  string
+}
+
```

Comment:
Please rename the struct to Spec and also the filename spec.go

Reply:
done